packages/modal/stories/modal.stories.tsx

```diff
@@ -1,6 +1,9 @@
 import { useDisclosure } from "@chakra-ui/hooks"
 import { chakra } from "@chakra-ui/system"
 import * as React from "react"
+//@ts-ignore
```

Comment:
I know you didn't add this, but would you mind removing it? Shouldn't be necessary now.

Reply:
Oh okay yeah I'll remove it.

Reply:
Thank you! Will get this merged today